engine/src/org/pentaho/di/job/entries/sendnagiospassivecheck/JobEntrySendNagiosPassiveCheck.java

```diff
@@ -548,6 +550,18 @@ public Result execute( Result previousResult, int nr ) {
     return result;
   }
 
+  @VisibleForTesting
+  protected String resolvePassword( String password ) {
```

Comment:
So much duplication. Please move this method into Utils class as a static helper method.